src/test/java/com/google/api/codegen/testdata/simplecompute_config.baseline

```diff
@@ -41,8 +41,8 @@ interfaces:
   # The entity_name is the name to be used as a basis for generated methods and
   # classes.
   collections:
-  - name_pattern: \{project}/regions/{region}/addresses
-    entity_name: region
+  - name_pattern: \{project}/regions/{region}
+    entity_name: addressesRegion
```

Comment:
It seems like this should just be `region`

Reply:
I started qualifying all the entity_names because there were a couple examples like `insertAddress(AddressName address, Address address)`, which has that variable name clash.

Reply:
Ok, this is changed back to `region`. 
Updated as so: 
- Methods that use the same canonical resource path will use the same resource entity; e.g. compute.router.insert and compute.address.insert both have paths `"{project}/project/{region}/region"` so they both use `RegionName`.
- In the case where multiple canonical paths have the same simple resource identifiers, then and only then is the `entity_name` qualified. e.g. method 
`compute.regionOperations.get`
has path
`"{project}/regions/{region}/operations/{operation}"`
and method
`"compute.globalOperations.get"`
has path
`"{project}/global/operations/{operation}"`
The qualified resource name classes `RegionOperationsOperationName` and `GlobalOperationsOperationName` are generated.